src/scripts/main.js

```diff
@@ -1,3 +1,143 @@
-'use strict';
 
-// write your code here
+import PlayingField from './playinfField';
+window.PlayingField = PlayingField;
+
+const playingField = new PlayingField();
+
+function startGame() {
+  playingField.field = [
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+  ];
+  playingField.score = 0;
+  playingField.start();
+  draw();
+  document.querySelector('.message.message-lose').classList.add('hidden');
+  document.querySelector('.message.message-win').classList.add('hidden');
+  document.querySelector('.start').classList.add('hidden');
+  document.querySelector('.restart').classList.remove('hidden');
+}
+
+document.querySelector('.start').addEventListener('click', function() {
+  document.querySelector('.message.message-start').classList.add('hidden');
+
+  startGame();
+});
+
+document.querySelector('.message.message-lose').addEventListener('click', function() {
+  startGame();
+});
+
+document.querySelector('.restart').addEventListener('click', function() {
+  startGame();
+});
+
+const gameKeys = function(occurrent) {
+  const loseMessage = document.querySelector('.message.message-lose');
+  const winMassage = document.querySelector('.message.message-win');
+
+  if (loseMessage && !(loseMessage.classList.contains('hidden'))) {
+    return;
+  }
+
+  if (winMassage && !(winMassage.classList.contains('hidden'))) {
+
+  } else {
```

Comment:
remove unused if. In this case better use NOT`!` operator
```suggestion
  if (winMassage && !(winMassage.classList.contains('hidden'))) {
 ????
  } else {
```